sender6.c

```diff
@@ -109,6 +114,40 @@ struct packet * sender6_send_icmp(struct sender6 *s, /* {{{ */
 	return NULL;
 } /* }}} */
 
+struct packet * sender6_send_tcp(struct sender6 *s, struct libnet_in6_addr dst,
```

Comment:
I guess this is fine to merge, but maybe add a `TODO` about allowing users to set the TCP checksum (we can then add a 2-byte payload like we do for ICMP). Also, add a TODO about letting users pass in TCP options (we do not want to build options ourselves, but we could take in a `u8 *buf` that the user builds and contains the options).